packages/components/tabs/src/Tabs.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import type { CommonProps } from '@contentful/f36-core';
+
+import { getTabsStyles } from './Tabs.styles';
+export interface TabsProps extends CommonProps {
+  variant?: 'default' | 'horizontal' | 'vertical';
```

Comment:
I don't think that the naming of the options is correct. If `variant` is `vertical`, it doesn't make tabs vertical, it just adds a vertical divider.